R/IDateTime.R

```diff
@@ -211,6 +211,12 @@ as.character.ITime = format.ITime = function(x, ...) {
 }
 
 as.data.frame.ITime = function(x, ...) {
+  nm = names(get("x", envir=parent.frame()))
```

Comment:
This assumes `as.data.frame()` is being called from `data.frame()`/`data.table()`, right? But it breaks direct call:

```r
as.data.frame(as.ITime('12:00:00'))
# Error in get("x", envir = parent.frame()) : object 'x' not found
```

I wonder if the ggplot2 issue cited is even still relevant...